source/opt/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_library(SpvToolsOpt
```

Comment:
This library name is inconsistent with the name of the main library (SPIRV-Tools)
I'm tempted to say this library should be SPIRV-Tools-opt.

But users might reasonably expect that SPIRV-Tools contains everything from the SPIRV-Tools project, but really it's the "core" stuff like the binary parser, assembler, etc.  But maybe that's ok.   We have the same issue with Shaderc and solved that problem somewhat with a way to build a shaderc_combined library.